### DISCUSSION TITLE
DIA-NN cannot load an Orbitrap Astral data file

### DISCUSSION THREAD

**Discussion by Decleo** [vdemichev/DiaNN#973] [Q&A]
Hello,

I am currently trying to process a DIA .raw file obtained from an Orbitrap Astral. It usually works just fine for files that are generated by an Exploris 480. One of the main differences seem to reside in the file size : about 2-3 GB for the Exploris 480, while the file size for the Orbitrap Astral is around 15 GB. I was wondering if it could explain the crash.

Here is the log  : 

diann.exe --f "D:\DIA-NN\Developpement\Astral\Hela03.raw
" --lib "D:\DIA-NN\Fasta and speclib\Human_UP000005640_20240108-lib.predicted.speclib" --threads 24 --verbose 1 --out "D:\DIA-NN\Developpement\Astral\Test_HeLa_Astral_1.tsv" --qvalue 0.01 --matrices  --no-nn --relaxed-prot-inf --smart-profiling --peak-center --no-ifs-removal 
DIA-NN 1.8.1 (Data-Independent Acquisition by Neural Networks)
Compiled on Apr 14 2022 15:31:19
Current date and time: Wed Mar 13 12:47:25 2024
CPU: GenuineIntel Intel(R) Xeon(R) Gold 5118 CPU @ 2.30GHz
SIMD instructions: AVX AVX2 AVX512CD AVX512F FMA SSE4.1 SSE4.2 
Logical CPU cores: 48
Thread number set to 24
Output will be filtered at 0.01 FDR
Precursor/protein x samples expression level matrices will be saved along with the main report
Neural network classifier disabled
Highly heuristic protein grouping will be used, to reduce the number of protein groups obtained; this mode is recommended for benchmarking protein ID numbers; use with caution for anything else
When generating a spectral library, in silico predicted spectra will be retained if deemed more reliable than experimental ones
Fixed-width center of each elution peak will be used for quantification
Interference removal from fragment elution curves disabled
DIA-NN will optimise the mass accuracy automatically using the first run in the experiment. This is useful primarily for quick initial analyses, when it is not yet known which mass accuracy setting works best for a particular acquisition scheme.

1 files will be processed
[0:00] Loading spectral library D:\DIA-NN\Fasta and speclib\Human_UP000005640_20240108-lib.predicted.speclib
[0:19] Library annotated with sequence database(s): D:\DIA-NN\fasta\uniprotkb_proteome_UP000005640_AND_revi_2024_01_08.fasta
[0:19] Gene names missing for some isoforms
[0:19] Library contains 20400 proteins, and 20198 genes
[0:24] Spectral library loaded: 20400 protein isoforms, 30630 protein groups and 10441107 precursors in 4125488 elution groups.
[0:24] Initialising library

[0:34] File #1/1
[0:34] Loading run D:\DIA-NN\Developpement\Astral\Hela03.raw

DIA-NN exited
DIA-NN-plotter.exe "D:\DIA-NN\Developpement\Astral\Test_HeLa_Astral_1.stats.tsv" "D:\DIA-NN\Developpement\Astral\Test_HeLa_Astral_1.tsv" "D:\DIA-NN\Developpement\Astral\Test_HeLa_Astral_1.pdf"
PDF report will be generated in the background

I tried to twitch the number of threads used by the software (1, 12, 23 and 24 threads) but it makes no difference.

The processing computer has the following specifications : 
Intel(R) Xeon(R) Gold 5118 CPU @ 2.30GHz   2.29 GHz  (2 processors)
128 GB (128 GB usable)
64-bit operating system, x64-based processor

Thank you in advance for your help.

**Comment by vdemichev** [comment]
I heard reports of DIA-NN not loading certain Astral files, not sure what's the reason. 
We definitely have it working OK on my PC on the Astral files we have looked at so far. 
The workaround however is to convert to .mzML using the settings recommended in DIA-NN docs.
You can also check just in case that Exploris files are loaded just fine on the specific PC - if not, need to install MSFileReader by the link in DIA-NN docs.

Best,
Vadim

**Comment by YanXu0213** [comment]
Hi,

Have you solved the problem? I had the same issue.

Best,
Yan

**Reply by Decleo** [reply]
Hi,
As recommended by Vadim, you can convert the .raw file into an .mzML using MSConvert. It worked just fine for me.
Best,
Léonie

**Reply by YanXu0213** [reply]
Thanks. I'll try.

**Reply by anamdv** [reply]
Hi!

I am having similar issues with Astral data in version 1.9, but in may case, even the mzml converted in MSConvert (with the parameters specified in DIANN docs) crashed... and when DIANN loads it, it exits without error message... Same raw files, when analyzed using the version 1.8.2 beta 27 did no give any problem). Any ideas?

**Reply by vdemichev** [reply]
So what happens if you try to process the .raw file (not mzML)? Provided MSFileReader is installed using the link here: https://github.com/vdemichev/DiaNN?tab=readme-ov-file#raw-data-formats

**Reply by vdemichev** [reply]
If the error occurs, can you please share the .raw file (a single one)? I will likely be able to troubleshoot this

**Reply by anamdv** [reply]
Yes, the same occurs when I use .raw.

I am reprocessing the data from PXD046444 (https://www.ebi.ac.uk/pride/archive/projects/PXD046444) and, for instance, this file: https://ftp.pride.ebi.ac.uk/pride/data/archive/2024/02/PXD046444/20230324_OLEP08_800ng_30min_E20H50Y30_180K_2Th3p5ms_03.raw gives this error, while others from that same submission are totally fine.


Thanks in advance for the help 😄

**Comment by vdemichev** [comment]
Thank you for spotting this! I've looked at what is happening and apparently occurs due to some incompatibility (bug) between the MSToolkit library DIA-NN uses to read .raw and .mzML files and the way instrument name is encoded in that .rae file. This causes an error in the MSToolkit library. I wonder if this only manifests with early data acquired before Astral release and maybe the associated acquisition software update? However the issue with those old data is easy for me to fix, i.e. I will include a fix of this in an update version release.

Best,
Vadim
